data/FilterList.json

```diff
@@ -6431,13 +6431,13 @@
     "emailAddress": null,
     "forumUrl": null,
     "homeUrl": "https://github.com/KonoromiHimaries/PolishSubFilters",
-    "issuesUrl": null,
+    "issuesUrl": "https://discord.gg/UEWEBqz",
```

Comment:
discord would be better suited for `chatUrl` rather than `issuesUrl`.

please update the PR, and then I'll merge.

thanks!